drivers/sensors/tmp112.c

```diff
@@ -282,40 +282,7 @@ int tmp112_register(FAR const char *devpath, uint8_t devnum,
     }
 
   priv->i2c = i2c;
-  switch (devnum)
```

Comment:
why add in the first patch and remove here, let's modify the first patch instead